YoutubeExplode/Bridge/PlaylistExtractor.cs

```diff
@@ -106,60 +141,70 @@ internal partial class PlaylistExtractor
-            .ElementAtOrNull(0)?
-            .GetPropertyOrNull("tabRenderer")?
-            .GetPropertyOrNull("content")?
-            .GetPropertyOrNull("sectionListRenderer")?
-            .GetPropertyOrNull("contents")?
-            .EnumerateArrayOrNull()?
-            .ElementAtOrNull(0)?
-            .GetPropertyOrNull("itemSectionRenderer")?
-            .GetPropertyOrNull("contents")?
-            .EnumerateArrayOrNull()?
-            .ElementAtOrNull(0)?
-            .GetPropertyOrNull("playlistVideoListRenderer")?
-            .GetPropertyOrNull("contents") ??
-
-        // Continuation response
-        _content
-            .GetPropertyOrNull("onResponseReceivedActions")?
-            .EnumerateArrayOrNull()?
-            .ElementAtOrNull(0)?
-            .GetPropertyOrNull("appendContinuationItemsAction")?
-            .GetPropertyOrNull("continuationItems")
+    public string? TryGetPlayListUrl() => Memo.Cache(this, () =>
+     TryGetPlaylistProperty()?
+         .GetPropertyOrNull("playlistShareUrl")?
+         .GetStringOrNull()
     );
 
     public IReadOnlyList<PlaylistVideoExtractor> GetVideos() => Memo.Cache(this, () =>
-        TryGetContentRoot()?
+        TryGetPlaylistProperty()?
+            .GetPropertyOrNull("contents")?
             .EnumerateArrayOrNull()?
-            .Select(j => j.GetPropertyOrNull("playlistVideoRenderer"))
+            .Select(j => j.GetPropertyOrNull("playlistPanelVideoRenderer"))
             .WhereNotNull()
             .Select(j => new PlaylistVideoExtractor(j))
             .ToArray() ??
 
         Array.Empty<PlaylistVideoExtractor>()
     );
 
+    public string? TryGetLastVideoId() => Memo.Cache(this, () =>
+        GetVideos()?
+            .LastOrDefault()?
+            .TryGetVideoId()
+    );
+
+    public int? TryGetLastIndex() => Memo.Cache(this, () =>
+        TryGetPlaylistProperty()?
+            .GetPropertyOrNull("contents")?
+            .EnumerateArrayOrNull()?
+            .Select(j => j.GetPropertyOrNull("playlistPanelVideoRenderer"))
+            .LastOrDefault()?
+            .GetPropertyOrNull("navigationEndpoint")?
+            .GetPropertyOrNull("watchEndpoint")?
+            .GetPropertyOrNull("index")?
+            .GetInt32OrNull()
+    );
+
     public string? TryGetContinuationToken() => Memo.Cache(this, () =>
-        TryGetContentRoot()?
+        TryGetPlaylistProperty()?
+            .GetPropertyOrNull("continuations")?
             .EnumerateArrayOrNull()?
-            .Select(j => j.GetPropertyOrNull("continuationItemRenderer"))
+            .Select(j => j.GetPropertyOrNull("nextContinuationData"))
             .WhereNotNull()
-            .FirstOrNull()?
-            .GetPropertyOrNull("continuationEndpoint")?
-            .GetPropertyOrNull("continuationCommand")?
-            .GetPropertyOrNull("token")?
+            .FirstOrDefault()
+            .GetPropertyOrNull("continuation")?
+            .GetStringOrNull()
+    );
```

Comment:
Are we actually passing this anywhere now?

Reply:
Not really, so i could remove this method  as we don't use it anymore

Reply:
Yes, please remove any code that is not longer used in this new flow